src/annotator/anchoring/pdf.js

```diff
@@ -447,56 +480,53 @@ export function anchor(root, selectors) {
  *
  * @param {HTMLElement} root - The root element
  * @param {Range} range
+ * @return {Promise<Selector[]>}
```

Comment:
This doesn't return a `Promise` anymore, does it?

Reply:
Both `anchor` and `describe` return promises as they may need to wait for page text to be fetched in order to generate and resolve selectors.